express/blocks/submit-email/submit-email.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+function validEmail(email) {
+  const emailRegex = /\S+@\S+\.\S+/;
+  return emailRegex.test(email);
+}
+
+function decorateSubmitEmailBlock($block) {
+  const $container = document.querySelector('.submit-email-container');
+
+  const $icon = $container.querySelector('.icon-creative-cloud-express, .icon-cc-express-logo');
+
+  // Remove submit-email block wrapping div
+  const $blockParentDiv = $block.parentElement;
+  $container.insertBefore($block, $container.firstChild);
+  $container.removeChild($blockParentDiv);
+  $icon.setAttribute('src', '/express/icons/creative-cloud-express.svg');
```

Comment:
Why do you need to reset the icon src?

Reply:
See my comment above, super confused why different images would have been coming back but they were 🤔

> This confused me.. sometimes the image was icon-creative-cloud-express and sometimes it was icon-cc-express-logo?